tkg/client/upgrade_addon.go

```diff
@@ -248,6 +248,13 @@ func (c *TkgClient) DoUpgradeAddon(regionalClusterClient clusterclient.Client, /
 			return errors.Wrap(err, "unable to get cluster configuration")
 		}
 
+		// ensure current kapp-controller deployment has last-applied annotation, which is required for future apply operations
+		if addonName == "addons-management/kapp-controller" {
+			if err = clusterClient.PatchKappControllerLastAppliedAnnotation(options.Namespace); err != nil {
```

Comment:
Should be `err :=` here (create and assign a local var). Otherwise, it's a "sloppy assignment" code style lint warning and generally a bad practice.

Reply:
Updated, thanks